engine/graphics/src/graphics.cpp

```diff
@@ -275,7 +293,19 @@ namespace dmGraphics
                 }
             }
         }
-        assert(selected_shader);
+
+        if (selected_shader == 0)
+        {
+            const char* error_hint = "";
+            if (language == ShaderDesc::LANGUAGE_SPIRV)
+            {
+                error_hint = "Has the project been built with spir-v output enabled?";
+            }
+
+            dmLogError("Unable to get a valid shader with shader language \"%s\" from a ShaderDesc for this context. %s",
+                GetShaderProgramLanguageLiteral(language), error_hint);
+        }
```

Comment:
Avoid the assert here and instead provide a better error string when we can't find a shader.